pkgs/development/python-modules/pycddl/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, pythonOlder
+, fetchPypi
+, buildPythonPackage
+, rustPlatform
+, pytestCheckHook
+, pytest-benchmark
+, psutil
+, cbor2
+}:
+
+buildPythonPackage rec {
+  pname = "pycddl";
+  version = "0.4.0";
```

Comment:
The latest version on PyPI remains 0.4.0 though I see there is something related to 0.4.1 in the upstream git repository.

Since the project does PyPI releases it seems like following PyPI makes more sense than grabbing tagged revisions directly from git.

Reply:
We don't take sdists to serious because they often miss test files.

Reply:
Can you clarify, please?  Are you saying this is fine as is or that it will only be acceptable if I switch to the unreleased 0.4.1 tag?

Reply:
From discussion on the dev channel, this means the upstream gitlab resources for the most recent release tag should be preferred if there are practical improvements that result (eg, the test suite being available).

The pycddl sdist on PyPI includes the test suite (indeed, a lot of the Nix expression here is _disabling_ parts of the test suite) and the v0.4.1 release has no meaningful user-facing changes compared to v0.4.0 so I think it makes sense to stick with v0.4.0 from PyPI.